distribution/lib/Standard/Table/0.0.0-dev/src/Excel/Excel_Range.enso

```diff
@@ -162,3 +168,21 @@ type Excel_Range
 
         top_valid <| bottom_valid <|
             Excel_Range.Value (Java_Range.forRows sheet top bottom)
+
+    ## ICON data_input
+       Convert an index (1-based) to a column name
+    column_name_for_index index:Integer -> Text =
+        if index < 0 || index > excel_2007_column_limit then Error.throw (Illegal_Argument.Error "Invalid column index.") else
```

Comment:
```suggestion
        if index <= 0 || index > excel_2007_column_limit then Error.throw (Illegal_Argument.Error "Invalid column index.") else
```

If it's 1-based then I guess `0` is also not a valid value?

Reply:
Also here I guess including the index in the error message may make it easier to spot mistakes.